boards/renesas/da1469x_dk_pro/dts/da1469x_dk_pro_lcdc.overlay

```diff
@@ -31,6 +31,14 @@
             bias-pull-up;
         };
     };
+
+    i2c2_sleep: i2c2_sleep {
```

Comment:
Where is this pin control entry being used?

Reply:
That entry should be used by the I2C driver (#72185). Here, the default pins definitions are overwritten to avoid conflicts with the LCD controller's signals which are routed to fixed pins.

Reply:
Ah, ok, thank you for the clarification.